test/utils/run-fixture.js

```diff
@@ -16,13 +18,9 @@ module.exports = (fixture) => {
   describe(fixture.describe, () => {
     // run with DEBUG=true to output coverage information to
     // terminal; this is useful when writing new tests.
-    if (process.env.DEBUG === 'true') {
-      it('outputs debug information', () => {
-        console.info('------------------')
-        console.info(JSON.stringify(coverageIstanbul, null, 2))
-        console.info('------------------')
-      })
-    }
+    it('outputs debug information', () => {
+      t.matchSnapshot(coverageIstanbul)
+    })
 
     fixture.assertions.forEach(assertion => {
```

Comment:
I don't think we need any of these assertions any longer, if I properly understand how the snapshot works?